### PR TITLE
#165026938 Configure application to use GraphQL

### PR DESCRIPTION
**Pull request by RonKbS**
#### What does this PR do?
 - It configures the application to allow use of graphQL

#### Description of Task to be completed?
The task requires us to be able to use graphQL queries and mutations.

#### How can this be tested manually?
- Check out to `ch-configure-graphql-165026938`.
- Start the app and run the following query;
```
query{
  users{
    id
    email
    phoneNumber
    isActive
  }
}
```
- Results returned will be similar to those shown in the screenshot

#### Screenshots
Screenshot 2019-04-04 at 11 06 28

#### Context
Using the insonmia app to run the query mentioned above

#### Pivotal Tracker Story
- [#165026938](https://www.pivotaltracker.com/story/show/165026938)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,12 +1,28 @@
...
+graphql-core==2.1
+graphql-relay==0.4.5
+gunicorn==19.9.0
+promise==2.2.1
 psycopg2==2.7.7
-dj-database-url==0.5.0
-django-heroku==0.3.1
+PyJWT==1.7.1
+python-dateutil==2.8.0
+pytz==2018.9
```
Comment (kimbugp) [inline requirements.txt:23]: Did you install all these packages?

**Comment by kimbugp** [comment]
Please update the piplock and pipfile. 
Run` pipenv install <package to be added> `

**Review by kimbugp — CHANGES_REQUESTED** [review]

**Comment by kimbugp** [comment]
Q

File: requirements.txt (hunk 1 above)
Comment (RonKbS) [reply requirements.txt:23]: Yes I did.

**Review by kimbugp — APPROVED** [review]

**Pull request merged**